pyiron_base/jobs/functioncontainer.py

```diff
@@ -0,0 +1,14 @@
+from pyiron_base.jobs.job.generic import GenericJob
+
+
+class FunctionContainer(GenericJob):
```

Comment:
Do you mean `PythonTemplateJob`?

Reply:
The issue is that `DataContainer` cannot store projects, that is why I cannot use the `PythonTemplateJob`.

Reply:
What is missing to store projects in `DataContainer` and how does this sidestep the problem?  I don't think `Project` comes with a `to_hdf`, but that also seems easy to add (just save the path).